firebase-inappmessaging-display/firebase-inappmessaging-display.gradle

```diff
@@ -80,10 +80,10 @@ dependencies {
     implementation "com.squareup.picasso:picasso:2.71828"
     implementation "com.squareup.okhttp:okhttp:2.7.5"
 
-    implementation "com.google.dagger:dagger:2.22"
-    implementation "com.google.dagger:dagger-android-support:2.22"
+    implementation "com.google.dagger:dagger:2.24"
+    implementation "com.google.dagger:dagger-android-support:2.24"
```

Comment:
It seems like this dep is never really used. Can we get rid of it?

Reply:
gone now 🗡